forum/qa-plugin/ckeditor4/plugins/syntaxhighlight/lang/pl.js

```diff
@@ -5,18 +5,7 @@ CKEDITOR.plugins.setLang('syntaxhighlight', 'pl', {
 	langLbl:'Wybierz język',
 	sourceTextareaEmptyError:'Kod źródłowy nie może być pusty!',
 	advancedTab:'Zaawansowane',
-	hideGutter:'Ukryj język',
-	hideGutterLbl:'Ukryj język i numery linii',
-	hideControls:'Ukryj informacje',
-	hideControlsLbl:'Ukryj dodatkowe informacje na górze bloczka z kodem.',
-	collapse:'Blokowanie',
-	collapseLbl:'Domyślnie zablokuj blok kodu (dodatkowe informacje muszą być włączone',
 	codeTitleLbl:'Tytuł bloku',
-	showColumns:'Pokaż kolumny',
-	showColumnsLbl:'Pokaż rząd kolumn w pierwszej linii',
-	lineWrap:'Wyłącz liniowe zawijanie bloku',
-	lineWrapLbl:'Wyłącz liniowe zawijanie bloku',
-	lineCount:'Domyślna liczba linii',
```

Comment:
nie potrzeba tych tłumaczeń?

Reply:
To zdaje się też zostało usunięte w PRze 202.

Odpowiadając na pytanie - opcje dla tych tłumaczeń usunąłem po konsultacji z @awaluk , bo nie są w ogóle lub były rzadko używane na produkcji.

Reply:
>lub były rzadko używane na produkcji.
To oznacza, że były używane.

Reply:
Rzadko, ale się zdarzało, tylko co z tego? Były, ale skoro bardzo sporadycznie i mogły psuć inne rzeczy, więc nie widziałem problemu w tym, aby je usunąć. Coś z tym jest nie tak?